shell/src/main/java/alluxio/cli/fsadmin/metrics/ClearCommand.java

```diff
@@ -0,0 +1,264 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.metrics;
+
+import alluxio.cli.CommandUtils;
+import alluxio.cli.fsadmin.command.AbstractFsAdminCommand;
+import alluxio.cli.fsadmin.command.Context;
+import alluxio.client.block.AlluxioBlockStore;
+import alluxio.client.block.BlockWorkerInfo;
+import alluxio.client.block.stream.BlockWorkerClient;
+import alluxio.client.file.FileSystemContext;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.grpc.ClearMetricsRequest;
+import alluxio.util.ThreadFactoryUtils;
+import alluxio.wire.WorkerNetAddress;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.stream.Collectors;
+
+/**
+ * Clear the leading master (and workers) metrics.
+ */
+public final class ClearCommand extends AbstractFsAdminCommand {
+  private static final String MASTER_OPTION_NAME = "master";
+  private static final String WORKERS_OPTION_NAME = "workers";
+  private static final String SPECIFIED_OPTION_NAME = "node";
```

Comment:
I think this option might be confusing, since it is only workers. Ideally, an option like this would be called `--workers=a,b,c`, but there is already an option for `--workers`. Also, I'm not entirely sure we will need a clear-specific-workers functionality just yet. We could just stick with `--workers` and in the future, see if we can overload `--workers` to also accept a list of individual hostnames.

What do you think?

Reply:
Our current option system does not support `--workers a,b,c` and `--workers` at the same time. I also feel the option names are confusing. 

If users curious about all the metrics during a certain period, they may want to clear all the master and workers metrics. I cannot think of a senario that users want to clear all the workers metrics without clearing the leading master metrics. There may be no need for clearing all the workers metrics without clearing the leading master metrics. (no need for --workers flag)

For --master, user may only curious about the cluster metrics, so it makes sense to clear the metrics of leading master only.

I would prefer to keep `--workers a,b,c` and remove the `--workers` flag.
So the new usage would be:
by default, clear the metrics of the whole cluster.
if `--master`, clear the metrics of the leading master
if `--workers a,b,c`, clear the metrics of specific worker.